modules/core/src/main/java/org/apache/ignite/internal/visor/cache/VisorCacheMetrics.java

```diff
@@ -55,6 +55,12 @@
     /** Gets number of keys in the cache, possibly with {@code null} values. */
     private int keySize;
 
+    /** Number of non-{@code null} values in the cache. */
```

Comment:
Maybe to add to comment "as a long type"?

Reply:
Done.